docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: '2'
+services:
+  calypso:
+    image: node:6.9.1
+    environment:
+      CALYPSO_ENV: 'development'
+    ports:
+      - '3000:3000'
+    volumes:
+      - ./:/calypso
+      - node_modules:/calypso/node_modules
```

Comment:
@withinboredom I'm not sure I understand what you are talking about with the named volume. to me it seems like we are telling Docker to find the directory named "node_modules" in some path resolution and then mount that inside the container at `/calypso/node_modules` - am I wrong here?